### DISCUSSION TITLE
Modal 内容滚动位置重置

### DISCUSSION THREAD

**Discussion by JohnNiang** [vbenjs/vue-vben-admin#255] [Ideas]
希望能够在 Modal 组件中提供一个重置滚动位置的功能，避免在数据变动后再次打开 Modal 仍旧停留在中间位置。 如下图所示：

![image](https://user-images.githubusercontent.com/16865714/107139079-64d7d300-6953-11eb-88ad-8a36664cadc6.png)

**Comment by anncwb** [comment]
我还没去测，目前应该是不管有没有销毁都会保持原来的位置。

**Comment by anncwb** [comment]
你要的重置滚动条是打开默认重置把？

**Comment by anncwb** [comment]
modal新增 scrollTop，默认true。打开会滚动到顶部

**Comment by JohnNiang** [comment]
> modal新增 scrollTop，默认true。打开会滚动到顶部

非常感谢您的适配。
